test/test_cases/run_tests.sh

```diff
@@ -27,7 +27,7 @@ do
       announce "Running '$filename'."
       ./test_case_setup.sh
       $filename
-      ./test_case_teardown.sh
+      ../../common/teardown_resources.sh
```

Comment:
i have a feeling that the common files should be in the parent directory. this way we won't need these changes

Reply:
I agree with you here